notebooks/115-async-api/115-async-api.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Runtime Inference Optimization with Async API\n",
+    "This notebook demostrates how to use [asynchronous execution](https://docs.openvino.ai/nightly/openvino_docs_deployment_optimization_guide_common.html) to improve data pipelining. \n",
+    "\n",
+    "OpenVINO™ Runtime supports inference in either synchronous or asynchronous mode. Using the Async API can improve application’s overall frame-rate, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. "
```

Comment:
```suggestion
    "OpenVINO Runtime supports inference in either synchronous or asynchronous mode. The key advantage of the Async API is that when a device is busy with inference, the application can perform other tasks in parallel (e.g. populating inputs or scheduling other requests) rather than wait for the current inference to complete first. "
```